ts/webui/src/components/managementExp/experimentConst.ts

```diff
@@ -0,0 +1,37 @@
+import { IDatePickerStrings } from '@fluentui/react';
+const DayPickerStrings: IDatePickerStrings = {
```

Comment:
Is this necessary? The doc says `strings` is optional.

Reply:
Thanks for your suggestion. It's optional, i had delete related code.